report_xls/__openerp__.py

```diff
@@ -26,8 +26,7 @@
     'author': "Noviat,Odoo Community Association (OCA)",
     'website': 'http://www.noviat.com',
     'category': 'Reporting',
-    'description': """
-Excel report engine
+    'description': """Excel report engine
```

Comment:
In this instance there isn't a ReadMe, and I was trying to keep the change-set minimal as this isn't a full conversion & I am just trying to fix the build. If this is a block however, I will move this over to a ReadMe using OCA template.